Development.md

```diff
@@ -0,0 +1,235 @@
+
+## Local development
+
+Currently the monorepo can be installed and built using **pnpm** except `web-staking` that has yet to be integrated and needs to be managed with **npm** (https://www.pivotaltracker.com/story/show/187887214)
+
+[NX](https://nx.dev/features/run-tasks) can be used for local script management, currently for every app except `web-staking`
+
+Run the different apps by their commands in the root's `package.json` like
+
+- `pnpm cli` for running the CLI
+- `pnpm web` for running sentry-xai.games locally
+- `pnpm desktop` for running the desktop app locally
+
+For local release building and signing SSL.com access is requires. Currently the credentials are set in the github repositories sentry & sentry-development, however, sentry.development is currently breaking on sign.
+
+A local release and a github release can be created without signing the release.
+Currently only the windows release gets signed.
+
+---------------------------------
+
+# Deployments & Releases
+
+## Develop Environment
+
+### Operator & Web-Connect
+
+Push a **tag** on `xai-foundation/sentry-development:working-branch` to trigger the github actions. This will deploy
+
+- **web-connect** to https://xai-foundation.github.io/sentry-development/ (github pages)
+- **CLI & Desktop App (all OS)** to https://github.com/xai-foundation/sentry-development/releases
+
+THIS WILL TRIGGER THE AUTO UPDATE FOR ALL PREVIOUS DESKTOP CLIENTS !
+
+**Important sepolia testnet updates:**
+- `packages\core\src\config.ts` needs to be updated, current sepolia testnet config can be found here https://github.com/xai-foundation/sentry-development/blob/build-sepolia/packages/core/src/config.ts
+- `useBlockIp()` hook in `apps/web-connect` needs to set `setBlocked(false);` for any user to disable geo-block
+- currently we still need to comment out hardcoded checks for chainIds and RPCs in web-connect and core. For this we can use the prepared branch `xai-foundation/sentry-development:build-sepolia`. For local testing and test deploys new branches can be created off `build-sepolia` and tags pushed from there.
+- remove `compareWithCDN` in  `packages\core\src\operator\operatorRuntime.ts`, there is no sepolia public node that publishes to the CDN
+
+### Staking App
+
+Push to `xai-foundation/sentry-development:develop` to trigger the cloud deploy of 
+
+- develop.app.xai.games
+- https://console.cloud.google.com/cloud-build/builds?hl=de&project=xai-games Source: `xai-foundation/sentry-development` (Google Cloud access required)
+
+
+Enabling sepolia testnet for the staking app is managed through environment variables set by the services team. The staking app has both sets of contract addresses and manages the main network through the env variable `NEXT_PUBLIC_APP_ENV=development`. This enables connecting the arbitrum sepolia network and disables the geo-block for the staking app.
```

Comment:
We should walk them through the steps?

1. Use the sample.env to create  starter .env file.
2. Set the NEXT_PUBLIC_APP_ENV=development in the .env
3. Set the MONGODB_URL=mongodb://127.0.0.1:27017/xai-app
    *note: if you have not "prepopulated" your local mongoDb, you need to do that next.
    * Prepopulate mongoDb by starting the CLI and running the sync-staking-pools command.
    * Ensure the current config file in packages/core/src/config.json is the Sepolia config file.

Reply:
Added the stept to the section

Reply:
Nice work!